### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Update README.md
##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,21 @@ Also for some notes specific to this collection see [our CONTRIBUTING documentat
...
+## Collection maintenance
+
+To learn how to maintain / become a maintainer of this collection, refer to:
+
+* [Committer guidelines](https://github.com/ansible-collections/community.general/blob/main/commit-rights.md).
+* [Maintainer guidelines](https://github.com/ansible-collections/community.mysql/blob/main/MAINTAINING.md).
+
+It is necessary for maintainers of this collection to be subscribed to:
+
+* The collection itself (the `Watch` button -> `All Activity` in the upper right corner of the repository's homepage).
```
Comment (felixfontein) [inline README.md:88]:
```suggestion
* The collection itself (the `Watch` button → `All Activity` in the upper right corner of the repository's homepage).
```

File: README.md [hunk 2]
```diff
@@ -76,7 +76,21 @@ Also for some notes specific to this collection see [our CONTRIBUTING documentat
 
 See [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#testing-collections).
 
-### Communication
+## Collection maintenance
+
+To learn how to maintain / become a maintainer of this collection, refer to:
+
+* [Committer guidelines](https://github.com/ansible-collections/community.general/blob/main/commit-rights.md).
+* [Maintainer guidelines](https://github.com/ansible-collections/community.mysql/blob/main/MAINTAINING.md).
```
Comment (felixfontein) [inline README.md:84]: Do you mean https://github.com/ansible/community-docs/blob/main/maintaining.rst, or the file in community.mysql?

Comment (Andersson007) [reply README.md]: I wish community.mysql would be a keystone collection but in this case i meant the maintaining.rst file:) thanks for spotting! fixed. Ready for review:)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@Andersson007 thanks for this improvement!

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing and merging!

**Pull request merged**